Logit.lua

```diff
@@ -305,5 +316,5 @@ end
 
 return setmetatable(Logit, {
   __call = Logit.new,
-  __index = Logit__index
+  __index = Logit__index,
```

Comment:
I saw all the changes in this file, and I think they're not so much significative and isn't the way how I like to code... But still this pull request adds support for some good things (mainly the LSP support), so it's ok :+1:

Reply:
I apologize for the automatic style formatting usage (stylua). I don't force you to use it. Sometimes it helps to fix unreadable code, but your code reads fine. The main fixes there are in the `local` keyword. For example, function `Logit__index` was global.

Reply:
> I apologize for the automatic style formatting usage (stylua). I don't force you to use it. Sometimes it helps to fix unreadable code, but your code reads fine. The main fixes there are in the `local` keyword. For example, function `Logit__index` was global.

Oohh so that's the `.stylua.toml` thing, makes sense